WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/MainState.kt

```diff
@@ -105,8 +102,7 @@ class MainState(
         playbackStateMutatorMutex.mutate {
             when (playbackState) {
                 is PlaybackState.Ready,
-                PlaybackState.PlayingVoice,
-                PlaybackState.PlayingMusic ->
+                PlaybackState.PlayingVoice ->
```

Comment:
MainState should be probably updated to a ViewModel and stop using MutatorMutex. Just put the state object in a MutableStateFlow.

But for another task.